### PR TITLE
Make contracts 4.24 compatible

### PR DESCRIPTION
**Pull request by artiebits**
# 🚀 Description

- [x] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](../blob/master/CONTRIBUTING.md)
- [ ] ✅ I've added tests where applicable to test my new functionality.
- [ ] 📖 I've made sure that my contracts are well-documented.
- [x] 🎨 I've run the JS/Solidity linters and fixed any issues (`npm run lint:all:fix`).

Truffle updated solc to 0.4.24 as well.

Warnings fixed:
- `keccak256` only accepts a single "bytes" argument. 
- Using contract member "balance" inherited from the address type is deprecated, use "address(contract).balance" instead.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "ganache-cli": "6.1.0",
     "solidity-coverage": "^0.5.0",
     "solium": "^1.1.7",
-    "truffle": "^4.1.8",
+    "truffle": "^4.1.9",
```
Comment (elopio) [inline package.json]: This should be 4.1.11:
https://github.com/trufflesuite/truffle/releases/tag/v4.1.11

**Comment by elopio** [comment]
Sorry @artiebits, once more.
@shrugs @frangio can we land this next, so there are no more conflicts on the way?

**Comment by shrugs** [comment]
oops we did it again 😓 

I'll personally avoid merging any more PRs until this one is merged. sorry for the extra work, @artiebits

**Comment by axic** [comment]
Any chance merging this and #1002? It would simplify our life at the Solidity repo a lot :)

**Comment by shrugs** [comment]
@axic will do!

**Comment by shrugs** [comment]
Also it'd be cool if these sort of upgrades were done automatically by a bot in the future, since it's a pretty easily automatable (but human-checkable) operation

**Comment by axic** [comment]
Btw, people with write access to this repo should be able (unless specifically disabled by the owner) to push to @artiebits PRs.

**Comment by artiebits** [comment]
no worries, updated

**Comment by frangio** [comment]
>it'd be cool if these sort of upgrades were done automatically by a bot in the future
This should be possible using Solium's fix mode!

**Pull request merged**